themes/default/layouts/migrate/terraform.html

```diff
@@ -1,43 +1,19 @@
 {{ define "hero" }}
-    {{ partial "hero" (dict "title" "Migrating to Pulumi from Terraform") }}
+    {{ partial "hero" (dict "title" "Pulumi for Terraform Users") }}
 {{ end }}
 
 {{ define "main" }}
     <section id="overview" class="container mx-auto px-2 my-16">
         <div class="flex flex-wrap">
             <div class="w-full lg:w-1/2 p-6">
                 <div>
-                    <h2>Increase productivity with programming languages</h2>
-                    <p>Terraform uses a proprietary DSL called HCL to describe and provision infrastructure resources.</p>
+                    <h2>Increase scale and productivity with programming languages</h2>
                     <p>
-                        In Pulumi, you describe infrastructure using familiar languages and tools, providing productivity gains, better abstraction and reuse, and developer
-                        approachability &mdash; while still supporting robust infrastructure as code provisioning across many clouds.
+                        Pulumi and Terraform describe and provision infrastructure differently. Terraform uses a proprietary domain-specific language (DSL) called HashiCorp
+                        Configuration Language (HCL). Pulumi uses general purpose languages that better help you tame cloud complexity, accelerate development velocity, and improve
```

Comment:
Pulumi uses general purpose languages that better help you tame cloud complexity and accelerate development velocity. 

Let's just replace that sentence with this. I think the security claims are too great.

Reply:
Why are security claims too great?